### PR TITLE
Add PruneCluster for Leaflet

### PR DESCRIPTION
**Pull request by simon04**
Based on https://github.com/SINTEF-9012/PruneCluster/blob/47a3c7d44eb04a31098b8de256d26ba9d957b0c7/package.json

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/SINTEF-9012/PruneCluster
 * Official website (optional, not the repository): http://sintef-9012.github.io/PruneCluster/
 * NPM package url (optional): https://www.npmjs.com/package/prunecluster
 * License and its reference: MIT, https://github.com/SINTEF-9012/PruneCluster/blob/master/LICENSE
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 37
   - Count of stars: 329
   - Count of forks: 88
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week: 163
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] Doesn't already exist / has duplicate issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
 * [ ] The parent of the commit(s) in the PR is not older than 3 days.
 * [ ] Pull request is sent from a non-master branch with a meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/prunecluster/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "leaflet",
+    "marker",
+    "realtime"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/SINTEF-9012/PruneCluster.git"
+  },
+  "filename": "PruneCluster.js",
```
Comment (extend1994) [inline ajax/libs/prunecluster/package.json]: Let's use `PruneCluster.min.js`, cdnjs will minify the file :)

File: ajax/libs/prunecluster/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "npmName": "prunecluster",
```
Comment (extend1994) [inline ajax/libs/prunecluster/package.json]: Let's use git auto-update, where we can see more build versions
```js
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/SINTEF-9012/PruneCluster.git",
    "fileMap": [
      {
        "basePath": "dist",
        "files": [
          "**/!(*.d.ts)"
        ]
      }
    ]
  }
```

`**/!(*.d.ts)` is to exclude PruneCluster.d.ts which is not for browser use.

**Comment by simon04** [comment]
I'm abandoning this PR. Please feel free to resolve the final issues or close the PR.

**Comment by extend1994** [comment]
Thanks @simon04 and sorry for letting you wait so long.
Close it now, welcome anyone interested in this library to open another PR.

**Pull request closed without merging**